docs/recipes.rst

```diff
@@ -219,3 +219,43 @@ An example of then using this:
     print author_schema.dump(author).data
 
 This is inspired by functionality from ColanderAlchemy.
+
+Smart Nested Field
+======================================================
+If you want to avoid lazy loading a nested attribute, but you still need id, then use this trick.
```

Comment:
Also, I believe there needs to be an empty line between the header and this text in order to be rendered correctly.